frontend/src/utilities/errors.ts

```diff
@@ -32,5 +33,74 @@ registerResponseHandler(ResponseType.DisplayError, (responseData: Response) => {
 	};
 	const buttons = [okButton];
 
-	createDialog("Warning", "Editor error", data.description, buttons);
+	createDialog("Warning", data.title, data.description, buttons);
 });
+
+registerResponseHandler(ResponseType.DisplayPanic, (responseData: Response) => {
+	const data = responseData as DisplayPanic;
+
+	const reloadButton: TextButtonWidget = {
+		kind: "TextButton",
+		callback: async () => window.location.reload(),
+		props: { label: "Reload", emphasized: true, minWidth: 96 },
+	};
+	const copyErrorLogButton: TextButtonWidget = {
+		kind: "TextButton",
+		callback: async () => navigator.clipboard.writeText(getPanicDetails()),
+		props: { label: "Copy Error Log", emphasized: false, minWidth: 96 },
+	};
+	const reportOnGithubButton: TextButtonWidget = {
+		kind: "TextButton",
+		callback: async () => window.open(githubUrl(), "_blank"),
+		props: { label: "Report Bug", emphasized: false, minWidth: 96 },
+	};
+	const buttons = [reloadButton, copyErrorLogButton, reportOnGithubButton];
+
+	createDialog("Warning", data.title, data.description, buttons);
+});
+
+function githubUrl() {
+	const url = new URL("https://github.com/GraphiteEditor/Graphite/issues/new");
+
+	const body = `
```

Comment:
Maybe put this text in a `.md` or `template` file and import it?

Reply:
Do you know how I could import from an external file at build time? If I do this, I would like to put the file in `/.github/ISSUE_TEMPLATE/crash-report.yaml` that the build process would have to grab it from outside the `/frontend` directory. I don't want to add another webpack plugin as a dependency though. Maybe there's a built-in way?